src/native/corehost/apphost/apphost.windows.cpp

```diff
@@ -6,6 +6,7 @@
 #include "pal.h"
 #include "trace.h"
 #include "utils.h"
+#include <shellapi.h>
```

Comment:
Is this needed?

Reply:
It is needed for the 
```c
            // Open the URL in default browser
            ::ShellExecuteW(
```

The question is - why it was not needed before...

Reply:
On why this include was not needed when this was building in installer partition. - clr partition build defines `WIN32_LEAN_AND_MEAN` and installer partition does not.